apps/condo/domains/property/components/panels/Builder/MapConstructor.ts

```diff
@@ -836,37 +836,39 @@ class MapEdit extends MapView {
         this.notifyUpdater()
     }
 
-    public updateSection (section: BuildingSection): void {
+    public updateSection (section: BuildingSection, renameNextUnits = true): void {
```

Comment:
why it's not a default behavior? do we really need it with `renameNextUnits = false` ?

Reply:
I don’t know :) It’s written in other parts of the map constructor, I didn’t break this rule